public/src/app-bar.html

```diff
@@ -115,8 +115,17 @@
         });
       }
 
-      _refreshBalance(reg) {
-        this.balance = "ℂ" + (reg.status.userBalance || 0).toFixed(3);
+      _refreshBalance(reg, fetched) {
+        setInterval(() => {
+          this.balance = "ℂ" + this._truncateBalance($core.balance);
+        }, 1000);
+      }
```

Comment:
no need for delay here

Reply:
This is setInterval, not setTimeout.  But you're right.  Should set it first, and then repeatedly after that.